docs/source/fitting/fitfunctions/StretchedExpFT.rst

```diff
@@ -15,10 +15,9 @@ Description
 
 Provides the Fourier Transform of the Symmetrized Stretched Exponential Function
 
-.. math:: S(Q,E) = Height \int_{-\infty}^{\infty} dt/h \cdot e^{-i2\pi (E-Centre)t/h} \cdot e^{-|\frac{t}{Tau}|^{Beta}} )
+.. math:: S(Q,E) = Height \int_{-\infty}^{\infty} dt/h \cdot e^{-i2\pi (E-Centre)t/h} \cdot e^{-|\frac{t}{Tau}|^{Beta}}
 
-with :math:`h` Planck's constant. If the energy units of energy are micro-eV, then tau is expressed in pico-seconds. If E-units are micro-eV then
-tau is expressed in nano-seconds.
+with :math:`h` Planck's constant. If the energy units of energy are milli-eV, then tau is expressed in pico-seconds. If E-units are micro-eV then tau is expressed in nano-seconds.
```

Comment:
I agree - the units of planks constant are given here
https://github.com/mantidproject/mantid/blob/476f5f70974836a5dd1e2ed83ecd346a8c1b704d/Framework/PythonInterface/plugins/functions/StretchedExpFTHelper.py#L126
@robertapplin confirmed in debug that the variable  ` planck_constant ~ 4` .
Me and @thomashampson  checked this and we get the same answer!
The value of planks constant is roughly `4e-15 eV s = 4(1e-3 eV)(1e-12 s) = 4 meV ps`
Or equivalently `4e-15 eV s = 4(1e-6 eV)(1e-9 s) = 4 mueV ns`